src/js/common/schemaform/review/SubmitButtons.jsx

```diff
@@ -49,6 +51,21 @@ export default function SubmitButtons(props) {
         </div>
       </div>
     );
+  } else if (submission.status === 'throttledError') {
+    submitButton = (
+      <ProgressButton
+        onButtonClick={onSubmit}
+        buttonText="Submit Application"
+        buttonClass="usa-button-primary"/>
```

Comment:
Does it make sense to disable this button until they can submit again? I think we generally try to shy away from disabling buttons, but I figured I'd ask.

Reply:
I'm hesitant to add that for something that will probably be seen by very few people. It would be nice, though.